### DISCUSSION TITLE
Add leaderboard to Playlist

### DISCUSSION THREAD

**Discussion by Layendan** [ppy/osu#29937] [Ideas]
This could also be added to daily challenge, but adding a leaderboard to playlist maps should be the priority in my opinion.

Normal play, leaderboard appears on the left:
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/dc3e8d9b-7d72-49f1-953b-8147751f5dca">

Playlist and Daily challenge, no leaderboard on the left:
<img width="1552" alt="image" src="https://github.com/user-attachments/assets/99d06cca-c454-457c-aa43-5bcdac2ded67">

**Comment by Jinkku** [comment]
Honestly it's weird that playlists and daily challenge don't have a leaderboard display as solo and multi. I agree pls add leaderboard display to those two.
